Extensions/TileMap/JsExtension.js

```diff
@@ -1800,6 +1804,93 @@ module.exports = {
         }
       }
 
+      /**
+       * This is called to update the PIXI object on the scene editor, without reloading the tilemap.
+       */
+      updatePixiTileMap() {
+        // Get the tileset resource to use
+        const tilemapAtlasImage = this._associatedObjectConfiguration
+          .getProperties()
```

Comment:
When investigating for performance improvements, I was that this getProperties was not insignificant, so better do it once

Reply:
Agree!